cognite/client/_api/sequences.py

```diff
@@ -901,8 +904,12 @@ def insert(
                 >>> data = c.sequences.data.retrieve(id=2,start=0,end=10)
                 >>> c.sequences.data.insert(rows=data, id=1,column_external_ids=None)
         """
+
```

Comment:
Examples should reflect the rename to `columns`. Found quite some usage still: 
<img width="813" alt="image" src="https://github.com/cognitedata/cognite-sdk-python/assets/8521241/e2744193-ce70-4d42-96ca-918bb14a7605">